setup.py

```diff
@@ -15,8 +15,9 @@
     packages=['django_nameko_standalone'],
     platforms=['Linux'],
     install_requires=[
-        "Django==3.0.7",
-        "nameko==2.12.0"
+        "Django>=3.0.7,<=4.2.6",
```

Comment:
I'd leave the patch part out, i.e `Django>=3,<=4.2`, assuming they respect the semver.
Given this is a library, we don't want to not allow future patches of the same Django minor version.

In fact, I'd even leave the minor out, keeping only the major, especially for Django, as I know they won't break anything in the API.

Reply:
It makes sense! I updated to have only the major version number and now it contains only the "greater than or equal" on setup.py and requirements